plugins/modem-manager/fu-mm-device.c

```diff
@@ -316,7 +317,8 @@ fu_mm_device_probe_from_omodem(FuMmDevice *self, MMObject *omodem, GError **erro
 				    "no physdev set");
 		return FALSE;
 	}
-	fu_device_set_physical_id(FU_DEVICE(self), sysfs_path);
+	platform_id = g_path_get_basename(sysfs_path);
+	fu_device_set_physical_id(FU_DEVICE(self), platform_id);
```

Comment:
I don't think this is always going to be unique -- can you describe a bit what you're trying to do here?

Reply:
yes, if FU_DEVICE_PRIVATE_FLAG_REPLUG_MATCH_GUID is set on the device.